### PR TITLE
Adopt Contributors Covenant v1.4 Code of Conduct

### PR DESCRIPTION
**Pull request by CodeMonkeyLeet**
Signed-off-by: Simon Leet <simon.leet@microsoft.com>

### CODE REVIEW

File: samples/file-encryptor/testfile [hunk 1]
```diff
@@ -1,28 +1,15 @@
...
-Microsoft plans to release the Open Enclave SDK under the MIT license
+This is dummy text file consisting of only English words and punctuation:
+-------------------------------------------------------------------------
 
+Our. Fifth have saying two they're firmament, herb. Shall, set for. Open
+firmament greater creeping gently winged thing likeness abundantly fourth
+hath subdue. That multiply night. I third whose there i. Give. Morning.
+Over forth light us.
 
+Herb. Replenish. Firmament lesser their of our lesser fowl were beginning
```
Comment (johnkord) [inline samples/file-encryptor/testfile]: Is this randomly generated?

Comment (CodeMonkeyLeet) [reply samples/file-encryptor/testfile]: Yes :)

File: docs/CodeOfConduct.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# Open Enclave SDK Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to make participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and expression,
+level of experience, education, socio-economic status, nationality, personal
+appearance, race, religion, or sexual identity and orientation.
```
Comment (anakrish) [inline docs/CodeOfConduct.md]: What about _political affiliation_? Also the organization that some contributor works for?

Comment (CodeMonkeyLeet) [reply docs/CodeOfConduct.md]: I would leave those as upstream suggestions to the Contributor Covenant. There will always be additional differences that we will have failed to enumerate, and folks should take it in the spirit of creating a "harassment-free experience for everyone".

**Pull request merged**